### PR TITLE
Complete week 1 portfolio website

### PR DESCRIPTION
**Pull request by sdotpeng**
I completed the website and added basic texts and images. I will learn more to enhance the website in next week while doing second week project.

@IslandCow Sean, could you check this pull request please?

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,19 +1,83 @@
...
+
+<body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>My SPS Portfolio</h1>
```
Comment (IslandCow) [inline portfolio/src/main/webapp/index.html:13]: h1 aligns as a block element.  Is there styling from content that you want here?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,19 +1,83 @@
...
     </div>
-  </body>
-</html>
+
+    <!-- Introduction -->
+    <div id="content">
```
Comment (IslandCow) [inline portfolio/src/main/webapp/index.html:17]: What is this id for? You should only have one element with a given id on a page.  If you just want styling, this should be a `class` instead.

**Review by IslandCow — APPROVED** [review]
One comment on the use of ids.  You can merge it after you fix it.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (sdotpeng) [reply portfolio/src/main/webapp/index.html:17]: Yes, my intention was to use the class so that I can apply the same styling to multiply blocks.

**Pull request merged**